content/datasources/kafka.md

```diff
@@ -0,0 +1,31 @@
+# Collecting Data from Kafka
+
+## Scenario
+
+[Kafka](http://kafka.apache.org) is a highly distributed messaging system.
+
+You run Kafka as a messaging system and now want to send the messages into various other systems.
+
+Fluentd can setup to collect messages from Kafka. Applications include:
+
+1. Sending Kafka messages into HDFS for analysis
+2. Sending Kafka messages into Elasticsearch for analysis
+
```

Comment:
Writing options like "Fluentd has two way to consume kafka message, kafka-fluentd-consumer or fluent-plugin-kafka" is better.